parachain/pallets/inbound-queue/src/benchmarking/fixtures.rs

```diff
@@ -1,20 +1,22 @@
+// Generated, do not edit!
+// See ethereum client README.md for instructions to generate
 use hex_literal::hex;
 use snowbridge_beacon_primitives::CompactExecutionHeader;
 use snowbridge_core::inbound::{Log, Message, Proof};
 use sp_std::vec;
 
 pub struct InboundQueueTest {
-	pub execution_header: CompactExecutionHeader,
-	pub message: Message,
+    pub execution_header: CompactExecutionHeader,
+    pub message: Message,
 }
```

Comment:
Cool!  IIUC the `InboundQueueFixture` is also auto-generated based on some template file. I would like it's not limited to benchmark tests and also apply to unit/integration tests.

More context in https://github.com/Snowfork/snowbridge/pull/1115#discussion_r1460904896